plugin/pxr/maya/lib/usdMaya/CMakeLists.txt

```diff
@@ -121,6 +121,7 @@ pxr_register_test(testUsdExportAssembly
     COMMAND "${TEST_INSTALL_PREFIX}/tests/testUsdExportAssembly"
     TESTENV testUsdExportAssembly
     ENV ${TEST_ENV}
+    WIN_DLL_PATH "${PXR_USD_LOCATION}/plugin/usd;${PXR_USD_LOCATION}/lib;${PXR_USD_LOCATION}/bin;${TEST_INSTALL_PREFIX}/maya/lib;${CMAKE_INSTALL_PREFIX}/lib"
```

Comment:
See the env var PXR_USD_WINDOWS_DLL_PATH to these paths for the test. This will tell the Tf module to use those paths (instead of PATH) when loading the python module.